ci-operator/step-registry/rosa/terraform/aws/e2e/test/rosa-terraform-aws-e2e-test-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: rosa-terraform-aws-e2e-test
+  from: ocm-tf-e2e
+  grace_period: 10m
+  commands: rosa-terraform-aws-e2e-test-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 300Mi
+  timeout: 2h0m0s
+  env:
+  - name: GATEWAY_URL
+    default: "https://api.stage.openshift.com"
+  - name: TOKEN_URL
+    default: "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
```

Comment:
For OCM Integration/Staging/Production, the sso is always this production sso, I think we can hard code it in the  makefile.

Reply:
it's ok to leave it as a parameter. future use.